library/src/main/java/com/rockerhieu/emojicon/EmojiconGridFragment.java

```diff
@@ -66,8 +67,7 @@ public void onViewCreated(View view, Bundle savedInstanceState) {
             mData = People.DATA;
             mUseSystemDefault = false;
         } else {
-            Object[] o = (Object[]) getArguments().getSerializable("emojicons");
-            mData = Arrays.asList(o).toArray(new Emojicon[o.length]);
+            mData = (Emojicon[]) getArguments().getParcelableArray(EMOJICONS_KEY);
```

Comment:
this throws an exception

Caused by: java.lang.ClassCastException: android.os.Parcelable[] cannot be cast to com.rockerhieu.emojicon.emoji.Emojicon[]

it can be fixed as described here: http://stackoverflow.com/questions/3647432/how-to-pass-an-array-of-address-objects-to-an-other-acitvity